html/bleaching.py

```diff
@@ -0,0 +1,164 @@
+import bleach
+
+
+class Cleaner(bleach.Cleaner):  # pylint: disable=too-few-public-methods
+    """
+    This class will only help lowering the strictness level of `bleach.Cleaner` super class by redefining the safe
+    values we're currently using and the prohibited ones.
+    """
+    def __init__(self, strict=True):
+        """
+        This is an override to the original bleaching cleaner, it deals with two bleaching modes, the strict mode,
+        and the trusted mode.
+
+        - Strict Mode (default): Mainly used for rendering strings that are not required to have too much attributes
+          and tags and doesn't aloow the use of most of the styles attributes.
+        - Trusted mode: Mainly used to render content that marked as trusted.
+
+        :param strict: whether we should strictly process the text or not.
+        """
+        attributes = {
+            'a': ['href', 'title', 'target', 'rel'],
+            'span': ['style'],
+            'p': ['style'],
+            'ul': [],
+            'img': ['src', 'alt', 'width', 'height'],
+        }
+        tags = [
+            'a', 'b', 'blockquote', 'code', 'em', 'i', 'li', 'span',
+            'ol', 'strong', 'ul', 'p', 'img', 'pre', 'h3', 'h4', 'h5', 'h6'
+        ]
+        styles = ['font-family', 'text-align', 'color', 'text-decoration', 'padding-left', 'padding-right']
+
+        if not strict:
+            attributes.update({
+                'abbr': ['title'],
+            })
+            attributes['ul'].append('style')
+            attributes['img'].append('style')
+
+            tags += ['h1', 'h2', 'script', 'sub', 'sup', 'div', 'abbr', 'iframe']
+
+            styles += ['list-style-type', 'font-size', 'border-width', 'margin']
+
+        self.styles = styles
+        self.tags = tags
+        self.attributes = attributes
+
+        super(Cleaner, self).__init__(
+            tags=self.tags,
+            attributes=self.attributes,
+            styles=self.styles,
+            strip=True,
+        )
+
+
+class SanitizedText(object):  # pylint: disable=too-few-public-methods
+    """
+    This class is responsible for maintaining unsafe string values saved in the database. It returns a safe value of the
+    passed text and an unsafe value if requested.
```

Comment:
Did you write this just for this block? It's nice :)  But is it being used anywhere? All I can see is it being used in two templates as `{self.sanitized_data}`

Reply:
Nope, I was working on a similar problem last year that requires bleaching as well, but that was on a model field level and this is why it's so detailed. Maybe it's an overkill for this task, so we can get rid of everything not related.